doc/source/whatsnew/v0.21.0.txt

```diff
@@ -43,9 +43,10 @@ Backwards incompatible API changes
 
 - Support has been dropped for Python 3.4 (:issue:`15251`)
 - The Categorical constructor no longer accepts a scalar for the ``categories`` keyword. (:issue:`16022`)
-
 - Accessing a non-existent attribute on a closed :class:`HDFStore` will now
   raise an ``AttributeError`` rather than a ``ClosedFileError`` (:issue:`16301`)
+- Compression defaults in HDF stores now follow pytable standards. Default type is 'zlib' and default level is 0 (:issue:`15943`)
```

Comment:
The default is no compression and this is what it says? complevel=0 turns compression off.